Assets/AnkrSDK/Examples/Scripts/CustomSceneManager.cs

```diff
@@ -5,16 +5,6 @@ namespace AnkrSDK.Examples
 {
 	public class CustomSceneManager : MonoBehaviour
 	{
-		public void NextScene()
-		{
-			SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
```

Comment:
Did you change invoke on this function in ERC20Example and ERC721Example

Reply:
This function is not called, only load scene